.gitattributes

```diff
@@ -11,4 +11,4 @@
 *.png filter=lfs diff=lfs merge=lfs -text
 *.adicht filter=lfs diff=lfs merge=lfs -text
 *.stl filter=lfs diff=lfs merge=lfs -text
-*.dxf filter=lfs diff=lfs merge=lfs -text
+*.SLDPRT filter=lfs diff=lfs merge=lfs -text
```

Comment:
What is SLDPRT?

Reply:
Solid works file which can be edited.

Reply:
And it should be explained in the readme. It is not just for you to answer this question to me personally this one time, but for everyone to be able to find out when they use this documentation.